CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,78 @@
+# Ilios Code of Conduct
+
+Like the technical community as a whole, the Ilios team and community is made up
```

Comment:
I think given our strong education focus we can drop the line `Like the technical community as a whole` and just lead with `The Ilios team and community`.